<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManager</class>
 <widget class="QDialog" name="ProfileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>19</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Manage Profiles&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListView" name="profileList">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>54</y>
     <width>177</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="profileBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>159</y>
     <width>470</width>
     <height>140</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>430</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Use http proxy:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="httpProxyHostInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="httpProxyPortInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="useHttpProxyCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="useHttpsProxyCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Use https Proxy:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="httpsProxyHostInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="httpsProxyPortInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="useFtpProxyCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Use ftp proxy:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="ftpProxyHostInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="ftpProxyPortInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="sameProxyButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="exceptionsInput">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>103</y>
      <width>346</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>108</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>No proxy for:</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="profileManagerButtonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="newProfileButton">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>124</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProfileButton">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>124</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>130</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pm_client.qrc">:/icons/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>profileList</tabstop>
  <tabstop>newProfileButton</tabstop>
  <tabstop>deleteProfileButton</tabstop>
  <tabstop>profileManagerButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="pm_client.qrc"/>
 </resources>
 <connections/>
</ui>
